index.html

```diff
@@ -907,13 +906,17 @@
         <li>If the <a>push subscription</a> has already been <a>deactivated</a>, resolve
         <var>promise</var> with <code>false</code> and terminate these steps.
         </li>
-        <li>Make a request to the <a>push service</a> to <a>deactivate</a> the <a>push
-        subscription</a>.
-        </li>
-        <li>If it was not possible to access the <a>push service</a>, reject <var>promise</var>
-        with a "<a>NetworkError</a>" exception and terminate these steps.
+        <li>Run the following step in parallel:
+          <ol>
+            <li><a>Deactivate</a> the <a>push subscription</a>. The <a>user agent</a> MUST NOT
+            deliver any further <a>push messages</a> for the <a>push subscription</a>.
+            <p>If the <a>user agent</a> failed to request the <a>push service</a> to <a>deactivate</a>
+            the <a>push subscription</a>, for example because of network failures, it SHOULD retry
+            the request to the <a>push service</a> for a reasonable amount of time.
```

Comment:
Yes. My intention (with the "run the following step in parallel" sub-step) is that we hold the promise from resolving until the necessary work to stop messages from being delivered to the Service Worker has been completed, but not any longer than that (i.e. network activity to the push service).